client/components/purchase-detail/style.scss

```diff
@@ -125,3 +125,23 @@
 	color: darken( $gray, 20% );
 	font-size: 21px;
 }
+
+.purchase-detail__info {
+	flex: 1;
+	margin-left: 8px;
+	margin-top: 0;
+	max-width: 260px;
+	display: inline-block;
+
+	.purchase-detail__info-icon-container {
+		display: inline-block;
+		margin-top: 2px;
+		margin-right: 4px;
+		float: left;
+		height: 14px;
+	}
+
+	@include breakpoint( "<660px" ) {
```

Comment:
From our [css guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#media-queries) we do mobile first so we should flip this around to 
```
.purchase-detail__info {
    margin-top: 8px;
    @include breakpoint( ">660px" ) {
         margin-top: 0;
    }
}